test/unit/math/prim/prob/wishart_cholesky_test.cpp

```diff
@@ -81,7 +81,7 @@ TEST(ProbDistributionsWishartCholesky, dof_0) {
   MatrixXd L_Y = Y.llt().matrixL();
   MatrixXd L_S = Sigma.llt().matrixL();
 
-  unsigned int dof = std::numeric_limits<double>::quiet_NaN();
```

Comment:
This change was needed to fix the `wishart_cholesky` prim failure from [this earlier Jenkins run](https://jenkins.flatironinstitute.org/blue/organizations/jenkins/Stan%2FMath/detail/PR-2792/21/pipeline/198)

This was relying on undefined behavior of a `NaN` overflowing to 0 when assigned to an int, which looks like it didn't happen in that Jenkins run. Let me know if you'd prefer I open separate PR and issue for this